docs/pt/docs/fastapi-cli.md

```diff
@@ -0,0 +1,84 @@
+# FastAPI CLI
+
+**FastAPI CLI** é um programa `fastapi` por linha de comando que você pode usar para rodar sua app FastAPI, gerenciar seu projeto FastAPI e mais.
+
+Quando você instala o FastAPI (ex.: com `pip install fastapi`), isso inclui um pacote chamado `fastapi-cli`. Esse pacote disponibiliza o comando `fastapi` no terminal.
+
+Para rodar sua app FastAPI em desenvolvimento, você pode usar o comando `fastapi dev`:
```

Comment:
As a brazilian portuguese native speaker, this phrase fits better with this version:

Para rodar sua app FastAPI em desenvolvimento, você pode usar o comando `fastapi dev`:
```suggestion
Para rodar seu app FastAPI em desenvolvimento, você pode usar o comando `fastapi dev`:
```